website/docs/d/scc_posture_scans_summary.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scan_summary"
+description: |-
+  Get information about scans_summary
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scan_summary
+
+Review information of Security and Compliance Center posture scan summary.
```

Comment:
1. The hyperlink should be provided with []() syntax, as shown in the first comment. 
2. Delete test. in the https://test.cloud.ibm.com..... link